spec/tren_spec.cr

```diff
@@ -11,4 +12,13 @@ describe Tren do
   it "should overload method" do
     get_users("fatih", 2).should eq("select * from users where name = 'fatih' and age = 2")
   end
+
+  it "should create and use method" do
+    get_user_info("alper", "t").should eq("select * from users where name = 'alper' and name = 't'")
+  end
+
+  it "should overload method" do
```

Comment:
This also wrong b/c it doesn't overload but using another query from same file. It may be something like `it "should parse second query from a file"`.